src/index.js

```diff
@@ -221,6 +221,36 @@ const mini = (tree, {mini}) => {
   return bypass(tree);
 };
 
+const sortAttr = (tree, {rules: {sortAttr}}) => {
+  tree.map(node => {
+    if (node.attrs && Object.keys(node.attrs).length > 1 && sortAttr) {
+      var key = Object.keys(node.attrs)
```

Comment:
You might want to try using `reduce`

Reply:
will try that

Reply:
you meant `reduce` instead of `map` ? or for sorting ?

Reply:
no I was thinking about what would be after `sort` instead
```js
        const newAttrsObject = {};

        keys.forEach(k => {
          newAttrsObject[k] = node.attrs[k];
        });

        node.attrs = newAttrsObject;
```

Reply:
```js
node.attrs = keys.sort().reduce((prev, next) => {...}, {})
```

Reply:
@anikethsaha ping ?

Reply:
sorry, idk something might have went wrong I didn't get the notification for this. 

anyway, 

is there any advantage of this over the current one?

cause, we are doing the `sort` for both  so the next iteration will be of `n` times for both the cases.  right ?

Reply:
it has a shorter syntax and does the same thing. you can also put it in a separate method / utility for creating a new object

```js
keys?.sort(YouMethodSort).reduce((prev, next) => {...}, {})
```

Reply:
oh ok...yea it does look neat with that way. I will change it.

Reply:
@Scrum I extracted the code. Not sure how to include `reduce` here cause we need to create the new obj with sorted keys. 

Let me know if this is fine.

Reply:
@anikethsaha I've made some minor changes, please take a look. So it turned out that I ruled from github, please rename my commit ))

Reply:
> So it turned out that I ruled from github, please rename my commit ))

didn't get this.


Looks good though.  👍